platform/fabric/core/generic/channel.go

```diff
@@ -172,6 +177,27 @@ func newChannel(network *network, name string, quiet bool) (*channel, error) {
 	return c, nil
 }
 
+var defaultCacheSize = 100
+
+// cacheSizeFromConfig returns the KVS cache size from current configuration.
+// Returns defaultCacheSize, if no configuration found.
+// Returns an error and defaultCacheSize, if the loaded value from configuration is invalid (must be >= 0).
+func cacheSizeFromConfig(sp view2.ServiceProvider) (int, error) {
+	configService := view2.GetConfigService(sp)
```

Comment:
this would not work because we might have multiple fabric configurations. Use channel's config field to get that

Reply:
good catch! updated!